CONTRIBUTING.md

```diff
@@ -3,11 +3,7 @@
 ### Getting and building
 
 ### Assumed
- * A working GCC (on mac os x something like `xcode-select --install` will get you started)
```

Comment:
Isn't this still needed? (with s/GCC/C compiler/, since the mac is all clang now). CGO calls out to /usr/bin/cc instead of using the embedded plan 9 compiler.